src/stories/lib/markdown-docs.jsx

```diff
@@ -0,0 +1,13 @@
+// we disable this rule because this file is only meant to be processed by Storybook. It is not part of our npm / CDN bundle.
+// eslint-disable-next-line import/no-extraneous-dependencies, no-unused-vars
+import React from 'react';
+// several docs rely on `cc-notice`
```

Comment:
Bear in mind that this wasn't a problem before because our components were all loaded automatically when landing on any page of our storybook.

Now that stories are lazy loaded, components are too, so `cc-notice` is only available if you have already encountered a story where it is imported.

This is why we import it here.